build.gradle

```diff
@@ -45,6 +45,14 @@ dependencies {
     compile group: 'com.fasterxml.jackson.core',
             name: 'jackson-databind',
             version: '2.10.3'
+    compile group: 'org.springframework.data',
```

Comment:
Není tenhle balíček už součástí toho podtím?

Reply:
Je to dost možné

Reply:
upraveno